tests/functional/z_cluster/nodes/test_nodes_maintenance.py

```
@@ -610,3 +624,288 @@ def test_pdb_check_simultaneous_node_drains(
+        eligible_osd_nodes = [n for n in osd_nodes if n in worker_names]
+        assert eligible_osd_nodes, "No worker-only OSD nodes available for shutdown"
+
+        mon_nodes = set(get_mon_running_nodes())
+        shutdown_order = [n for n in eligible_osd_nodes if n not in mon_nodes] + [
+            n for n in eligible_osd_nodes if n in mon_nodes
+        ]
+
+        eligible_count = len(eligible_osd_nodes)
+        log.info(
+            f"Eligible worker-only OSD nodes for shutdown: {shutdown_order} "
+            f"({eligible_count} of {total_hosts} total OSD hosts)"
```

Ruff (0.15.20)

[warning] Logging statement uses f-string

(G004)